src/sourceMaps/sourceMap.ts

```diff
@@ -223,7 +224,7 @@ export class SourceMap {
             return null;
         } else {
             return {
-                line: position.line - 1, // Back to 0-indexed lines
+                line: global["evalSources"] && global["evalSources"].find(s => s == this._generatedPath) ? position.line : position.line - 1, // check if eval code else Back to 0-indexed lines
```

Comment:
Same here - the 0-indexed or 1-indexed lines are a property of the source-map lib, it doesn't matter whether the code originated as an eval script or not.